lib/io.toit

```diff
@@ -57,7 +57,7 @@ abstract class Writer:
   is-closed_/bool := false
   endian_/EndianWriter? := null
   byte-cache_/ByteArray? := null
-  consumed_/int := 0
+  processed_/int := 0
```

Comment:
You omitted the type in the reader. Maybe be consistent about this?

Reply:
done. (added type to reader).